### PR TITLE
Added docs about branches.

### PR DESCRIPTION
**Pull request by thoragudf**

### CODE REVIEW

**Comment by hugihlynsson** [comment]
#### 📝Changed routes:


| [**/workspace/branches**](https://docs-kbgqp5hon.vercel.app/workspace/branches) |
|:-:|
|<a href="https://docs-kbgqp5hon.vercel.app/workspace/branches">Screenshot of /workspace/branches</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/dX9_CdszDuK1wzYaT2MaS5b_nw8_.png">(view full size)</a> |







Commit 744f4553c59fc8c7a951969891569543df14c431 (https://docs-kbgqp5hon.vercel.app).

**Review by alex-tpom6oh — APPROVED** [review]
👍

File: pages/workspace/branches.mdx [hunk 1]
```diff
@@ -0,0 +1,48 @@
+---
+layout: 'guide'
+title: 'Branches'
+# abstract: 'Learn about branched workflows'
+---
+
+import Link from '../../components/Link';
+
+
```
Comment (hugihlynsson) [inline pages/workspace/branches.mdx:9]: I think that we can skip this
```suggestion
```

File: pages/workspace/branches.mdx [hunk 2]
```diff
@@ -0,0 +1,48 @@
+---
+layout: 'guide'
+title: 'Branches'
+# abstract: 'Learn about branched workflows'
```
Comment (hugihlynsson) [inline pages/workspace/branches.mdx:4]: How about something like this?
```suggestion
abstract: 'Seamlessly work together on multiple fronts using branches'
```

File: pages/workspace/branches.mdx [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+---
+
+import Link from '../../components/Link';
+
+
+## What are branches?
+
+Avo offers branched workflows similar to git, to enable seamless collaboration between and within teams in your company. That means when you are making changes to your tracking plan, you can do it on a branch such that you can play around with your changes without affecting the source of truth on the main branch.
```
Comment (hugihlynsson) [inline pages/workspace/branches.mdx:12]: Not to get all into Minto, but I'd love to start _Why_ here. Why would someone need branches? Perhaps something like this: 
```suggestion
In Avo you can collaborate in realtime on defining, expanding and implementing your tracking plan. But you might want to try out some changes or make sure that a change is implemented before you change your tracking plan source of truth. To capture this work in progress you can use branches.
```

**Review by hugihlynsson — APPROVED** [review]
This is a great start! I had some thought but we should just ship this fast and then iterate on it.

**Pull request merged**